examples/aggregator/gcp/aggregator-dashboard.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
@daviddrysdale I guess this is the closest to borgmon / viceroy that we can get with open source? Wonder if you know of any alternatives to make this a bit more declarative.

Reply:
Nope, 'fraid not.  I did some (very) minimal [Prometheus](https://github.com/google/certificate-transparency-go/blob/master/trillian/integration/consoles/reads.html) [consoles](https://github.com/google/certificate-transparency-go/blob/master/trillian/integration/consoles/writes.html) for [CT/Trillian](https://github.com/google/certificate-transparency-go/blob/master/trillian/docs/ManualDeployment.md#monitoring), but they looked pretty similar to this.